lib/buttons-and-links/KIconButton.vue

```diff
@@ -90,6 +91,13 @@
         type: String,
         default: null, // https://github.com/learningequality/kolibri-design-system/issues/168
       },
+      /**
+       * Tooltip position, passes to existing 'UiTooltip' prop for position
```

Comment:
Rather than "passes to existing 'UiTooltip' prop for position" comment, could we list values that get accepted? This will go to the documentation automatically. From the KDS user point of view, it doesn't matter that much what we use under the hood when compared to knowing what values will work with this prop. A nice example is `appearance` prop of `KExternalLink` https://design-system.learningequality.org/kexternallink/#prop:appearance